### DISCUSSION TITLE
Where is documentation for using proxies with HTTParty pls?

### DISCUSSION THREAD

**Discussion by JetsonDavis** [jnunemaker/httparty#763] [General]
I can't find any official documentation or documentation that is accurate!

**Comment by jnunemaker** [comment]
I'd search proxy in this file. 

https://github.com/jnunemaker/httparty/blob/45233c1d1572679c7feeecdd6f3b5348c5ffbee2/lib/httparty.rb#L39-L42

The options are noted there and the methods have similar names with some docs.
